.changeset/wet-parents-check.md

```diff
@@ -0,0 +1,6 @@
+---
+"@osdk/client": patch
+"@osdk/api": patch
+---
+
+Remove where clause from aggregate options.
```

Comment:
Can we say more here? like "as it was never actually doing anything" or whatever sounds good?